invenio_files_rest/models.py

```diff
@@ -1111,6 +1121,112 @@ def relink_all(cls, old_file, new_file):
             ObjectVersion.query.filter_by(file_id=str(old_file.id)).update({
                 ObjectVersion.file_id: str(new_file.id)})
 
+    def get_tags(self):
```

Comment:
I am not sure how to use the `association_proxy` ...
Looking at the docs it seems to be used in many-to-many relationships, whereas here we have a one-to-many.
I tried setting it up by it makes things quite complex adding some nasty circular dependencies between `ObjectVersion` and `ObjectVersionTag`